### ISSUE TITLE
Be able to set a site username and multiple accounts

### ISSUE THREAD

**Issue by damyo** [getmeli/meli#188]
I want my project to feel well protected when sharing the site with collaborators and clients.
However, it doesn't look professional for the user name to be `user`.
As if the administrator account ID is `admin`.

So I at least want to be able to set the user name myself.
And if possible, I would like to be able to set up multiple accounts like Netlify.
Netlify can configure multiple accounts through `_headers` as follows.
`Basic-Auth: user1:password1 user2:password2`

It's good enough now, but I hope you consider my opinion.

**Comment by gempain** [comment]
@damyo I agree that using `user` is not ideal and we can change this. Having multiple accounts also makes sense and doable quite easily as Caddy's API allows this already. We'll add this to the roadmap.

**Comment by gempain** [comment]
@damyo, when we developed the password protection, we intended it to be just a password, and used basic auth + hardcoded username `user` as a temporary method. In practice, we planned to replace the basic auth feature with an html form with a password input, so you shouldn't have to use a username at all. This was inspired from the way [Ghost](https://github.com/TryGhost/Ghost) does it. In that sense, the password feature was not intended to become something where you could create multiple accounts with username/password.

That said, we wanted to know a bit more about how you would use this feature. What problem does it solve for you ?

**Comment by Kim-Chuljoong** [comment]
@gempain Let's take an example. Some of the team members have Group A and Group B. Group A has permission to projects A-1, A-2 and A-3, and Group B has permission to projects B-1, B-2, and B-3. Group A and Group B use different IDs and passwords. However, I started a new C-1 project, and this project has both A and B groups. In this case, if I register both the ID and password of both groups in the settings, the problem will be solved. Also, in the case of Group C, the A-1 and B-1 projects have permission. Then similarly, if I add the C group's ID and password to that project in the settings it will work. In addition, I want users to clearly feel that they have or do not have permission to the project with their own ID and password.
I hope my explanation makes sense. The key is to flexibly match our projects and users.

**Comment by gempain** [comment]
So, why not using the same password for sites A1, A2 and A3, then another password for B1, B2 and B3, and when you start project C1, use another password that you can share with teams 1 and 2 ? It's not as you described but it solves your problem.

**Comment by Kim-Chuljoong** [comment]
If I can register and use multiple passwords, the problem is solved as you said.
Currently, meli is using an ID.
Also, Netlify, which I have experienced, can use multiple IDs and passwords.
That's why I proposed this method.
Personally, I want to be able to use both ID and password, but if you don't use an ID, that's great if you can apply it quickly.
